draft-barnes-mimi-arch.md

```diff
@@ -298,6 +305,34 @@ Client        Follower        Hub         Follower        Client
 ~~~~~
 {: #fig-protocols title="MIMI Protocols" }
 
+## End-to-End Security
+
+As noted above, all of the clients participating in a room are part of the same
+end-to-end security context.  This allows them to encrypt and integrity-protect
```

Comment:
What is meant by "integrity-protect" here that's not covered by authentication and agreement mentioned below?

Reply:
Good point.  It's also kind of unnecessarily wordy.